.github/workflows/publish-image-to-dockerhub.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           docker pull $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:${GITHUB_REF##*/}
+          docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:${GITHUB_REF##*/}
```

Comment:
But you added only tagging, not actual push step.

Reply:
Fixed)